code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -300,7 +300,7 @@
 	spawn_flags = SPECIES_CAN_JOIN
 
 /datum/species/diona
-	spawn_flags = SPECIES_CAN_JOIN
+	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED
```

Comment:
Also true, though it is kinda nice to already have the predefined 'hey, you wanna change something about diona for VS? do it here'